Dependency problem

Your project is not up to date with bunyan dependencies, I got this error.  I had to fix the version number of bunyan.

npm ERR! peerinvalid The package bunyan does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer bunyan-blackhole@1.0.1 wants bunyan@~1.3.4

Could you update ? 

Thanks

--- a/package.json
+++ b/package.json
@@ -25,7 +25,12 @@
   },
   "homepage": "https://github.com/Floby/node-bunyan-blackhole",
   "dependencies": {
-    "bunyan": "~1.3.4",
     "stream-blackhole": "^1.0.3"
+  },
+  "peerDependencies": {
+    "bunyan": "~1.x.x"
+  },
+  "devDependencies": {
+    "bunyan": "1.8.1"
   }
 }

